apps/roboshield/.env

```
@@ -9,3 +9,5 @@ SENTRY_PROJECT=roboshield
 
 NEXT_PUBLIC_IMAGE_DOMAINS="*.codeforafrica.org"
 NEXT_PUBLIC_IMAGE_UNOPTIMIZED="true"
+
+NEXT_PUBLIC_APP_URL=http://localhost:3000
```

```suggestion
NEXT_PUBLIC_APP_URL=http://localhost:3000
PAYLOAD_PUBLIC_APP_URL=$NEXT_PUBLIC_APP_URL
```

i.e. PAYLOAD should use NEXT URL if not set.